Polishchuk_Dariia_HW2(4 sum of odd numbers).cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+
+int main() {
+
+  size_t array_size{10};
+  int a[array_size];
+  constexpr int min{-60};
+  constexpr int max{90};
+  long sum{0};
+
+  std::cout << "Enter a number from " << min << " before " << max << std::endl;
+
+  for (int i = 0; i < array_size; ++i) {
+
+    while (true) {
+      std::cin >> a[i];
+      if (a[i] > min && a[i] < max) {
```

Comment:
```suggestion
      if (a[i] >= min && a[i] <= max) {
```
наверное min и max включительно?